src/components/MessageForm/MessageForm.tsx

```diff
@@ -31,7 +31,7 @@ export const MessageForm = ({
   }, [textFieldRef])
 
   const canMessageBeSent = () => {
-    return textMessage.trim().length > 0 && !isMessageSending
+    return textMessage.trim().length > 0 && textMessage.trim().length < 1000 && !isMessageSending
```

Comment:
It looks like this is different than what was shown in https://github.com/jeremyckahn/chitchatter/issues/6#issuecomment-1252692681. Was the correct branch pushed?